src/Arcus.Templates.Tests.Integration/Fixture/SelfSignedCertificate.cs

```diff
@@ -0,0 +1,198 @@
+﻿using System;
+using System.IO;
+using System.Security.Cryptography.X509Certificates;
+using GuardNet;
+using Org.BouncyCastle.Asn1.X509;
+using Org.BouncyCastle.Crypto;
+using Org.BouncyCastle.Crypto.Generators;
+using Org.BouncyCastle.Crypto.Operators;
+using Org.BouncyCastle.Crypto.Prng;
+using Org.BouncyCastle.Math;
+using Org.BouncyCastle.Pkcs;
+using Org.BouncyCastle.Security;
+using Org.BouncyCastle.Utilities;
+using Org.BouncyCastle.X509;
+using X509Certificate = Org.BouncyCastle.X509.X509Certificate;
+
+namespace Arcus.Templates.Tests.Integration.Fixture
+{
+    /// <summary>
+    /// Exposes several certificate generation function to control the OCSP and/or CRL information of the certificates.
+    /// </summary>
+    public static class SelfSignedCertificate
```

Comment:
Yep, is copy. If we would use `paket` instead of NuGet package reference tags, we could just reference the file from a GitHub repo.

Reply:
Don't get me wrong but don't really see why we cannot use a NuGet package because if you depend on another project you would want it to be stable and ready and not in flux, no?

Reply:
Feel free to create an issue where we can track this

Reply:
Yep

Reply:
Maybe https://github.com/arcus-azure/arcus is best location for this?